confidential-data-hub/kms/src/plugins/mod.rs

```diff
@@ -14,6 +14,9 @@ pub mod aliyun;
 
 pub mod kbs;
 
+#[cfg(feature = "ehsm")]
+pub mod ehsm;
+
 #[derive(AsRefStr, EnumString)]
 pub enum DecryptorProvider {
```

Comment:
We might also need to add `eHSM-KMS` to `new_decryptor`

Reply:
Will do.